kpi/010.adoc

```diff
@@ -1,13 +1,13 @@
-== L. Distribution information
+== KPI-{counter:kpi}: Distribution information
 
 Metadata records should contain information regarding how to access the data
 which it is describing.
 
 === Status
 
-Draft
+Approved
 
-=== WMCP element(s)
+=== WCMP element(s)
```

Comment:
line 39 in xml example: 

DT/DSI/OP/TEL is not an individual. Not critical, but this is a common BP that I don't want to perpetuate.